examples/hosted-login-code-flow/src/main/resources/application.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2017 Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+logging:
+  level:
+    root: INFO
+    org.springframework.web: DEBUG
+    org.springframework.security: DEBUG
+
+okta:
+  oauth2:
+    # default is unset
+    customLoginRoute: /login
+
+    # default is '/login'
+    redirectUri: /login/okta
+
+security:
+  oauth2:
+    client:
+      # Need to figure out how to set this, this needs to be an absolute URI
+      userAuthorizationUri: http://localhost:8080/login
```

Comment:
This is a Spring Security setting, or ours?

Reply:
Spring, it needs an absolute URI.  (Ideally we need a better solution for this)